### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by scientes**
**Description**

cleaned endpoints for swagger

**Tick the appropriate choice (Delete the rest):**

* [x] Bug Fix (Non- Breaking)

### CODE REVIEW

**Comment by scientes** [comment]
I hope i didn't miss anything

**Comment by lucasace** [comment]
One small thing, could you remove the background task header from in the `playing_11_post` function

The PR might break if I do it again

**Comment by scientes** [comment]
hmm i tested it locally and it worked fine

File: app/main.py [hunk 1]
```diff
@@ -64,10 +64,10 @@ async def home_post(match: str = Form(...)):
 
 
 @app.get("/playing11", response_class=HTMLResponse)
-def playing_11(request: Request):
+def playing_11(request: Request,team1,team2):
 
     squad1,squad2,file,match_type = cricket.get_squad_file_match_type(
-        [request.query_params["team1"], request.query_params["team2"]]
+        [team1, team2]]
```
Comment (lucasace) [inline app/main.py:70]: Extra bracket

**Review by lucasace — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 1 above)
Comment (lucasace) [inline app/main.py:70]: Theres a extra bracket here

**Comment by lucasace** [comment]
Works
Thanks for contributing! 😄

**Pull request merged**